src/os_posix/os_union_fs.c

```diff
@@ -973,15 +976,15 @@ __union_fs_open_file(WT_FILE_SYSTEM *fs, WT_SESSION *wt_session, const char *nam
     WT_UNUSED(readonly);
     WT_UNUSED(which);
 
-    // TODO: Handle WT_FS_OPEN_FILE_TYPE_DIRECTORY
+    /* TODO: Handle WT_FS_OPEN_FILE_TYPE_DIRECTORY */
```

Comment:
Can you drop: "FIXME-WT-1380" on here instead of the todo?

Reply:
Added: [9f8e50e](https://github.com/wiredtiger/wiredtiger/pull/11268/commits/9f8e50e9e9e25c96dd296661234032fd94c5c45c)